onnxruntime/core/providers/cpu/tensor/gather_elements.cc

```diff
@@ -159,7 +161,7 @@ static void core_impl(const Tensor* input_tensor, const Tensor* indices_tensor,
   int64_t output_counter = 0;
 
   auto conditional_batch_call = [ttp, inner_dim_size](std::function<void(ptrdiff_t)> f) {
-    if (inner_dim_size < 10 * 1000) {  // TODO: tune this, arbitrary threshold
+    if (inner_dim_size < kParallelizationThreshold) {  // TODO: tune this, arbitrary threshold
```

Comment:
I saw below this parallelization may be called multiple times from a loop. I was wondering if there was a way to call TryBatchParallelFor only once.

Reply:
In the model I was working on the inner loop is the biggest contributor to runtime. In fact the outer loop runs only once. Hence, I didn't rework this more than what was required.